gridspeccer/cli.py

```diff
@@ -47,6 +47,13 @@ def plot():
         type=str,
         default="../fig",
     )
+    parser.add_argument(
+        "--independent-plots-filetype",
+        help="filetype of independent plots. those are queried from additional"
+        "gridspecs that are then plotted independently",
+        type=str,
+        default="None",
+    )
```

Comment:
Would it be interesting to have another parameter supplying a format string that maps subfigure name to filename? Would allow pre-/suffixing all generated files.